tests/framework/extensions/namespaces/deletenamespaces.go

```diff
@@ -0,0 +1,7 @@
+package namespaces
+
+import v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
+
+func DeleteNamespace(namespaceID *v1.SteveAPIObject, steveclient *v1.Client) error {
```

Comment:
Public functions need their own godoc comment explanation. Could we add an explanation? Also, do we need this function to be an extension? It seems like it's a single-function wrapper